src/TrackRowInfo.js

```diff
@@ -29,6 +30,7 @@ export default function TrackRowInfo(props) {
         trackWidth, trackHeight, 
         rowInfo,
         transformedRowInfo, 
+        aggregatedRowInfo,
```

Comment:
Short response: Reflecting your comments, I removed `rowInfo` which is not being used anymore, and renamed `aggregatedRowInfo` to `rowInfo`.

Long response: `aggregatedRowInfo` is similar to `rowInfo`. The only difference is that elements of `aggregatedRowInfo` can be an array of JSON Object just like how `selectRows` represent the aggregated rows. This variable is used for showing data in filtering interfaces since we want to enable users to filtering rows based on the original data.

As I found `rowInfo` is not being  used anymore, I removed `rowInfo` and renamed the `aggregatedRowInfo` to `rowInfo` since this is less misleading (consistent to how we call `selectRows` no matter if it contains an array of indices). I also added more comments.